server/src/test/java/io/druid/client/CachingClusteredClientFunctionalityTest.java

```diff
@@ -0,0 +1,227 @@
+/*
+ * Druid - a distributed column store.
```

Comment:
I'm less concerned about the lack of MMX and more concerned about the lack of ANY copyright owner, which seems odd